README.rdoc

```diff
@@ -69,8 +73,30 @@ The plugin supports the following parameters:
              log message and the detection will start from scratch.
              This is intended as a safeguard against buffering very large
              amounts of data before outputting them to the log stream.
+             Zero means no limit.
              Default: 1000.
 
+[max_bytes]  Maximum number of bytes in a detected exception stack trace.
+             If this maximum number is exceeded, the detected exception stack
```

Comment:
"detected exception stack trace that has been detected so far" has one too many instances of "detected". I'd lose the first one.